internal/server/server.go

```diff
@@ -0,0 +1,24 @@
+package server
+
+import (
+	"common-go-example/internal/config"
+	"github.com/buaazp/fasthttprouter"
+	"github.com/kintohub/utils-go/logger"
+	"github.com/valyala/fasthttp"
+)
+
+func Start() {
+	r := newRouter()
+	logger.Infof("Starting server on port %v", config.ServerPort)
+	logger.Errorf("Fasthttp server crashed %v",
+		fasthttp.ListenAndServe(":"+config.ServerPort, r.Handler))
```

Comment:
the port should be passed as a parameter instead of loading here

Reply:
Good one.

Reply:
Loading?